docs/sources/administration/api-keys/index.md

```diff
@@ -45,30 +53,95 @@ This topic shows you how to create an API key using the Grafana UI. You can also
    - If you are unsure about how long an API key should be valid, we recommend that you choose a short duration, such as a few hours. This approach limits the risk of having API keys that are valid for a long time.
 1. Click **Add**.
 
-## Migrate API Keys to Grafana service accounts
+## Migrate API keys to Grafana service accounts
 
-You can migrate one or all API keys to [Grafana service accounts]({{< relref "../service-accounts/" >}}). When you migrate an API key to a service account, a service account will be created with a service account token.
-The API key will continue to work, and you can find it in the [Grafana service account tokens]({{< relref "../service-accounts/#service-account-tokens" >}}) details.
-For more information about benefits of service accounts, refer to [Grafana service account benefits]({{< relref "../service-accounts/#service-account-benefits" >}}).
+As an alternative to using API keys for authentication, you can use a service account-based authentication system. When compared to API keys, service accounts have limited scopes which provides more security than using API keys. For more information about the benefits of service accounts, refer to [Grafana service account benefits]({{< relref "../service-accounts/#service-account-benefits" >}}).
 
-You can choose to migrate a single API key or all API keys. Note that when you migrate all API keys, you can't create new API keys anymore and will have to use service accounts instead.
+The service account endpoints generate a machine-user for authentication instead of using API keys. When you migrate an API key to a service account, a service account will be created with a service account token.
 
-### Before you begin
+> **Note:** If you are using API keys for authentication, we recommend that you migrate your integration to the service account authentication method. The API key will continue to work. You can locate the API key in the [Grafana service account tokens]({{< relref "../service-accounts/#service-account-tokens" >}}) details.
 
-- Ensure you have permission to create Grafana service accounts. For more information about permissions, refer to [Roles and permissions]({{< relref "../roles-and-permissions/#" >}}).
+This section shows you how to migrate your integration to use the new service account endpoints. You can migrate your API keys using:
```

Comment:
```suggestion
This section shows you how to migrate your integration to use the new service account endpoints. To migrate your API keys, use:
```

Reply:
Use all of these? One or more of these? One of these?

Reply:
use all of these steps. i think maybe you are confused by the word use here? 

if we say "To migrate your API keys:" would this make more sense?

Reply:
this is really dependant on the customer. if they have API calls using API keys, or terraform using API keys they need to migrate through that section, does this make sense?